src/main/java/taxi/model/Driver.java

```diff
@@ -6,13 +6,38 @@
     private Long id;
     private String name;
     private String licenseNumber;
+    private String login;
+    private String password;
 
     public Driver() {
```

Comment:
Lets define toString() method to display driver instances properly in exception messages(be attentive to sensitive driver fields)